src/languages/graphql.js

```diff
@@ -62,16 +62,10 @@ export default function(hljs) {
       },
       {
         scope: "symbol",
-        begin: hljs.regex.concat(GQL_NAME, /\s*:/),
-        returnBegin: true,
-        contains: [
-          {
-            scope: "symbol",
-            begin: GQL_NAME,
-            end: /\s*:/,
-            excludeEnd: true
-          }
-        ],
+        begin: hljs.regex.lookahead(hljs.regex.concat(GQL_NAME, /\s*:/)),
+        includeBegin: true,
+        end: /\s*:/,
+        excludeEnd: true,
```

Comment:
```suggestion
        begin: regex.concat(GQL_NAME,regex.lookahead(/\s*:/)),
        end: regex.lookahead(/\s*:/),
```

excludeEnd is casually deprecated.  There is no `includeBegin`, that's just the default, but we shouldn't need it.  I think you want to match the name but not the delimiter, right?  And we should set a `const regex` to avoid repeating hljs. over and over since these are already long enough.

Reply:
I must be doing something wrong, if I did not have `excludeEnd` then the `:` gets picked up in the scope. if I didn't include it in the `end` matcher then it gets too greedy and picks up everything until it finds a `:`

Reply:
> then the : gets picked up in the scope

That's what look-ahead guards against, but once it's in look-ahead it doesn't need to be in end anymore... because begin is already matching it -just not consuming it.

All tests are green... any final thoughts? :)

Reply:
I'd like to go forward with this version and I can consider any updates for a future version, if you are OK with that